src/operator-docs/transformation/concatMap.ts

```diff
@@ -1,6 +1,78 @@
 import { OperatorDoc } from '../operator.model';
 
 export const concatMap: OperatorDoc = {
-  'name': 'concatMap',
-  'operatorType': 'transformation'
+  name: 'concatMap',
+  operatorType: 'transformation',
+  signature: `concatMap(project: (value: T, index: number) =>  ObservableInput<I>,
```

Comment:
I think we are going without generic signature, correct?

Reply:
I am not sure about it, I was based on the switchMap docs, https://github.com/ReactiveX/rxjs-docs/blob/master/src/operator-docs/transformation/switchMap.ts.

Reply:
I am changing to the not generic version. 👍